rune/libenclave/attestation/sgx/ias/ias.go

```diff
@@ -106,21 +106,20 @@ func NewIasAttestation(cfg map[string]string) (*IasAttestation, error) {
 func (ias *IasAttestation) CheckQuote(q []byte) error {
 	quote := (*intelsgx.Quote)(unsafe.Pointer(&q[0]))
 
-	logrus.Debugf("Target Platform's Quote")
-	logrus.Debugf("  Quote Body")
-	logrus.Debugf("    QUOTE Structure Version:                               %d",
+	logrus.Debugf("QUOTE:")
+	logrus.Debugf("  Version:                              			  %d\n",
 		quote.Version)
-	logrus.Debugf("    EPID Signature Type:                                   %d",
-		quote.SignatureType)
-	logrus.Debugf("    Platform's EPID Group ID:                              %#08x",
-		quote.Gid)
-	logrus.Debugf("    Quoting Enclave's ISV assigned SVN:                    %#04x",
-		quote.ISVSvnQe)
-	logrus.Debugf("    Provisioning Certification Enclave's ISV assigned SVN: %#04x",
-		quote.ISVSvnPce)
-	logrus.Debugf("    EPID Basename:                                         0x%v",
-		hex.EncodeToString(quote.Basename[:]))
-	logrus.Debugf("  Report Body")
+	logrus.Debugf("  Attestation Key Type:                 			  %d\n",
+		quote.AttestationKeyType)
+	logrus.Debugf("  Quoting Enclave SVN:                  			  %d\n",
+		quote.QeSvn)
+	logrus.Debugf("  PCE SVN:                              			  %d\n",
+		quote.PceSvn)
+	logrus.Debugf("  Quoting Enclave Vendor Id:            			  0x%v\n",
+		hex.EncodeToString(quote.QeVendorId[:]))
+	logrus.Debugf("  User Data:                            			  0x%v\n",
+		hex.EncodeToString(quote.UserData[:]))
+	logrus.Debugf("  Report:                               			  ...\n")
```

Comment:
Why you change the quote's structure in IAS specific code?

Reply:
Quote structure should be defined in a general package and attester/challenger/verifier can be imported